src/tableFormatter.ts

```diff
@@ -93,15 +111,19 @@ class MarkdownDocumentFormatter implements DocumentFormattingEditProvider {
         lines[1] = lines[1].map((cell, i) => {
             if (/:-+:/.test(cell)) {
                 //:---:
+                colWidth[i] = this.checkColWidth(colWidth[i], 5)
```

Comment:
I think we can just use `Math.max(colWidth[i], 5)`

Reply:
Screw me ... of course we can. Thanks for reviewing!